### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by Deepakdanger**
Here I have done testing of function which is returning something. 👍 
- Done testing of 'createProject' function.
- Done testing of 'createTodo' function.
 Here I am doing testing via jest. 😃

### CODE REVIEW

File: src/test/create.test.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import { createProject, createTodo } from '../create';
+
+describe('Create the project', () => {
```
Comment (rindrajosia) [inline src/test/create.test.js:3]: [REQUIRED] There are two ways to make test units positive and negative. Please add negative tests.

File: src/test/create.test.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    expect(project.name).toEqual('sleep');
+  });
+
+  test('it will return project task type ', () => {
+    const project = createProject('sleep');
+    expect(typeof project.todos).toEqual('object');
+  });
+});
+
+describe('Create a todo', () => {
```
Comment (rindrajosia) [inline src/test/create.test.js:20]: [REQUIRED] There are two ways to make test units positive and negative. Please add negative tests.

File: README.md [hunk 3]
```diff
@@ -40,7 +40,21 @@ npm install
...
-   
+
+### Testing
+ First to install Jest 
+ ```sh
+ npm install --save-dev jest
+```
+Include this in scripts of package.json.
+ ```sh
+"test": "jest"
```
Comment (rindrajosia) [inline README.md]: - [ ] [REQUIRED] As `jest` is already installed when user uses `npm install `, I think, this information is not necessary.

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Hi, 

Good job so far!
There are some issues to revise before continuing to the next project but hang in there and read the comments below with care. You can do it! :muscle:


## Required Changes ♻️

- [x] [REQUIRED] all functions should be tested. Please add tests for all functions inside project.js

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by Deepakdanger** [comment]
I have done all the unit tests. For testing the project.js file, I am stuck and I arrange a meeting with the senior reviewer and find out those testing is entering into a complicated integration test. so he said to do all unit tests and submit the project.

**Review by oloomoses — APPROVED** [review]
Hi @Deepakdanger, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

_Slack: Oloo Moses_
------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**